tests/models/animals/test_animal_communities.py

```diff
@@ -17,19 +17,41 @@ def functional_group_list_instance(shared_datadir):
 
 
 @pytest.fixture
-def animal_community_instance(functional_group_list_instance):
+def animal_model_instance(data_instance, functional_group_list_instance):
```

Comment:
If you use this fixture across multiple scripts just put a copy of it in `tests/conftest.py` and then it becomes accessible everywhere

Reply:
Note that you can also have `tests/models/animals/conftest.py` to share animal model specific fixtures and keep the root conftest clean.